tests/deepsparse/v2/integration_tests/test_llms.py

```diff
@@ -0,0 +1,358 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This test suite consumes config files to test the text generation pipeline
+for various scenarios.
+
+A sample config file is a yaml that requires the following fields:
+    cadence: The cadence of the tests. The available options are:
+              "nightly", "weekly" and "commit". By default, only
+              the tests that have cadence "commit" will be run
+              in GHA. This parameter can be both a string or a
+              list of strings.
+    model_path: The path to the model to be tested
+                (sparsezoo stub/hf model path/local_path)
+    torch_model_name: The name of the torch model
+                (to generate ground truth info)
+    prompt: The prompt to use for testing
+    precision: The precision for the logits/kv_cache entries
+        comparison
+    internal_kv_cache: The type of the internal KV cache
+        management. Is a list that can contain the following
+        values: [True], [False] or [True, False] (to test both
+        external and internal KV cache management)
+"""
+import os
+from typing import List, Tuple
+
+import numpy
+
+import pytest
+from deepsparse.transformers.pipelines.text_generation import TextGenerationOutput
+from deepsparse.v2.pipeline import Pipeline
+from deepsparse.v2.text_generation import TextGenerationPipeline
+from sparsezoo import Model
+from tests.deepsparse.transformers.pipelines.integration_tests.helpers import (
+    TorchGroundTruthSource,
+    parse_params,
+    validate_internal_kv_cache,
+)
+
+
+CONFIGS_DIRECTORY = "tests/deepsparse/v2/integration_tests/configs"
+
+
+@pytest.fixture()
+def max_new_tokens() -> int:
+    return 64
```

Comment:
is the fix to edit the typo between max_new_tokens and max_length in this PR? or has it already landed?